packages/api2/test/quantification.e2e-spec.ts

```diff
@@ -0,0 +1,240 @@
+
+      quantifications.push(await quantificationsSeeder.seedQuantification({
+        quantifier: adminUser._id,
+        score: 0,
+        scoreRealized: 30,
+        dismissed: false,
+        praise: praise._id,
+        createdAt: endDate,
+      }));
+
+      praises.push(praise);
+
+      praises.push(await praiseSeeder.seedPraise({
+        createdAt: dateBetween
+      }));
+
+      quantifications.push(await quantificationsSeeder.seedQuantification({
+        quantifier: adminUser._id,
+        score: 0,
+        scoreRealized: 20,
+        dismissed: false,
+        praise: praises[1]._id,
+        createdAt: dateBetween,
+      }));
+
+      praises.push(await praiseSeeder.seedPraise({
+        createdAt: startDate
+      }));
+
+      quantifications.push(await quantificationsSeeder.seedQuantification({
+        quantifier: adminUser._id,
+        score: 0,
+        scoreRealized: 10,
+        dismissed: false,
+        praise: praises[2]._id,
+        createdAt: startDate,
+      }));
+    });
+
+    test('401 when not authenticated', async () => {
+      await request(server).get('/quantifications/export').send().expect(401);
+    });
+
+    test('200 when authenticated', async () => {
+      await authorizedGetRequest(
+        `/quantifications/export?format=json&startDate=${dateBetween.toISOString()}&endDate=${endDate.toISOString()}`,
+        app,
+        adminUserAccessToken
+      ).expect(200);
+    });
+
+    test('returns quantifications that matches seeded list in json format, filtered by date', async () => {
+      const response = await authorizedGetRequest(
+        `/quantifications/export?format=json&startDate=${dateBetween.toISOString()}&endDate=${endDate.toISOString()}`,
+        app,
+        adminUserAccessToken,
+      ).expect(200);
+      // exclude quantifications from the startDate
+      expect(response.body.length).toBe(2);
+      for (const returnedQuantification of response.body) {
+        expect(
+          quantifications.some(
+            (createdQuantification) =>
+            String(createdQuantification._id) !== returnedQuantification._id,
+          ),
+          // eslint-disable-next-line jest-extended/prefer-to-be-true
+        ).toBe(true);
+      }
+    });
+
+    test('returns quantifications that matches seeded list in csv format, filtered by date', async () => {
+      const response = await authorizedGetRequest(
+        `/quantifications/export?format=csv&startDate=${dateBetween.toISOString()}&endDate=${endDate.toISOString()}`,
+        app,
+        adminUserAccessToken,
+      ).expect(200);
+      expect(response.text).toBeDefined();
+      expect(response.text).toContain('_id');
+      expect(response.text).toContain(String(quantifications[0].praise));
+      expect(response.text).toContain('score');
+      expect(response.text).toContain('scoreRealized');
+      expect(response.text).toContain('dismissed');
+      // expect(response.text).toContain('duplicatePraise');
```

Comment:
This column is excluded when parsing to CSV for some reason, but not sure what is the reason.